### PR TITLE
fix issue #3,#4,#5

### PR DESCRIPTION
**Pull request by wesleytao**
Add more description for rasa quick start

### CODE REVIEW

**Comment by harrywang** [comment]
@wesleytao see how you can refer to issues in comments using `#`: fix issues  #2 , #3,  #5

File: rasa_quick_starter/readme.md [hunk 1]
```diff
@@ -21,32 +23,130 @@ source venv/bin/activate
...
+### explaination
+ 1. **Checkpoint** it is used stored past models, as new functionality comes in we might add new data to train. check point help us manipulate among different old version
+
+2. **intent_classifer** is related to the features and models
+
+3. **policy** is how your robot react to the response, it might depend on different type.
+fallback_policy is when robot failed to predict your intent. memorization_policy is ask robot to strictly follow the storied.md
+kearas policy use keras to train the model (you could define architechture by yourself)  
 
+4. **domain.json** and **domain.yml** contain the same infomation when you defined in domain.yml in the first place
```
Comment (harrywang) [inline rasa_quick_starter/readme.md]: information - typo

Comment (wesleytao) [reply rasa_quick_starter/readme.md]: Thank you for correction. I am learning these GitHub features.

File: rasa_quick_starter/readme.md [hunk 2]
```diff
@@ -21,32 +23,130 @@ source venv/bin/activate
...
+5. **policy_metadata** store the information about evaluation of different policies, and ensemble method.
+
+
+# Run
 In the current folder, run the following commands
 
 - train core dialogue: `python -m rasa_core.train -d domain.yml -s stories.md -o models/dialogue`. tensorflow is used to train models and a new folder named `models` will be created after this command.
+
 - train NLU: `python -m rasa_nlu.train -c nlu_config.yml --data nlu.md -o models --fixed_model_name nlu --project current --verbose`， `current` folder is created under `model` after this command.
-- start the conversation: `python -m rasa_core.run --enable_api -d models/dialogue -u models/current/nlu`, now a web server has beens started (a flask server to serve APIs at port 5005) and you can interact with the bot using command line.
```
Comment (harrywang) [inline rasa_quick_starter/readme.md]: `--enable_api` is used in Leon's demo this morning - we should keep it unless you have a good reason not to.

**Pull request merged**